azure/services/disks/spec.go

```diff
@@ -0,0 +1,40 @@
+/*
```

Comment:
verify test is failing because this header doesn't match the expected header

> Boilerplate header is wrong for: /home/prow/go/src/sigs.k8s.io/cluster-api-provider-azure/azure/services/disks/spec.go